pandas/io/parsers/readers.py

```diff
@@ -262,9 +262,9 @@
     more strings (corresponding to the columns defined by `parse_dates`) as
     arguments.
 
-   .. deprecated:: 2.0.0
-      Use ``date_format`` instead, or read in as ``object`` and then apply
-      :func:`to_datetime` as-needed.
+    .. deprecated:: 2.0.0
```

Comment:
This is not rendering properly: https://pandas.pydata.org/docs/dev/reference/api/pandas.read_csv.html